### ISSUE TITLE
OVPsim free version shows license expired.

### ISSUE THREAD

**Issue by subhajit26** [riscv/riscv-ovpsim#27]
Hi,
        I installed ovpsim free version for some riscv based test . It was working till yesterday but today all of a sudden it shows license has expired and we need to renew it. Please see the messages below and provide some suggestion.

Fatal (PER_TMO) License has expired.
Please contact license@imperas.com to renew

**Comment by Imperas** [comment]
Yes riscvOVPsim has now moved - the information is on the main [README](https://github.com/riscv/riscv-ovpsim/blob/master/README.md).
Please visit its new home: [github.com/riscv-ovpsim](https://github.com/riscv-ovpsim/imperas-riscv-tests),
and the enhanced one on:  https://www.ovpworld.org/riscvOVPsimPlus/
